Include the mingw directory in the PATH as this file is only used
for cross-compilation.

Add the j flag to tar as all downloaded files are .tar.bz2.

--- tasks/bin/cross-ruby.rake.orig	Wed Dec 31 16:00:00 1969
+++ tasks/bin/cross-ruby.rake	Wed Nov 30 04:23:04 2011
@@ -17,6 +17,8 @@
 # for cross-compilation.
 #
 
+ENV['PATH'] = "${LOCALBASE}/mingw32/bin:#{ENV['PATH']}"
+
 require 'rake'
 require 'rake/clean'
 
@@ -84,7 +86,7 @@ end
 source_file = RUBY_SOURCE ? RUBY_SOURCE.split('/').last : "#{RUBY_CC_VERSION}.tar.bz2"
 file "#{USER_HOME}/sources/#{RUBY_CC_VERSION}" => ["#{USER_HOME}/sources/#{source_file}"] do |t|
   chdir File.dirname(t.name) do
-    t.prerequisites.each { |f| sh "tar xf #{File.basename(f)}" }
+    t.prerequisites.each { |f| sh "tar jxf #{File.basename(f)}" }
   end
 end
 
